### PR TITLE
[2159] updated links and emails to postman.com

### LINKED ISSUE #2159: Link in README 404s
The link text "getpostman" in the main README leads to a 404. (highlighted below)

Screen Shot 2020-02-03 at 9 35 36 PM

Also, we should update that text since we are no longer using the 'getpostman' domain name.

### PR DESCRIPTION
**Pull request by tristandenyer**
This fixes #2159 and few other things:
— fixed 404 link to our website
— updated link text on above
— updated help@ email to use postman.com
— updated community link to use postman.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The built site will only host the most up-to-date docs. All legacy documentation
 * [Postman Integrations](https://www.postman.com/integrations/)
 * [The Postman API](https://docs.api.getpostman.com/)
 * [Postman Release Notes](https://www.postman.com/downloads/release-notes)
```
Comment (DannyDainton) [inline README.md]: @tristandenyer / @SueSmith We seem to have 2 release note links in this list :)

File: legacy-documentation/v5/billing_and_payment/pricing-policy.md [hunk 2]
```diff
@@ -42,27 +42,27 @@ We won't delete any of your data or collections. Only the creators will have acc
 
 ### We can't pay by credit card. Can you invoice my company?
 
-We can invoice large teams who subscribe to a yearly plan. Contact us at [help@getpostman.com][0] to arrange this. All monthly plans must be paid by credit card.
+We can invoice large teams who subscribe to a yearly plan. Contact us at [help@postman.com][0] to arrange this. All monthly plans must be paid by credit card.
 
 ### I'm not the billing contact for our company. Can I have billing statements automatically sent to someone else?
 
-You can set up a billing contact email address when you get started on Pro. If you have already gotten started and not put that in, you can email us at [help@getpostman.com][0].
+You can set up a billing contact email address when you get started on Pro. If you have already gotten started and not put that in, you can email us at [help@postman.com][0].
```
Comment (DannyDainton) [inline legacy-documentation/v5/billing_and_payment/pricing-policy.md]: This mentions `Pro` which isn't a plan anymore. Should we reword the sentence or change to `Team`?  Also, `gotten` might be too much of a localised word for people to understand. :)

File: legacy-documentation/v5/postman/launching_postman/syncing.md [hunk 3]
```diff
@@ -59,6 +59,6 @@ When you reload the app, Postman automatically retrieves the most recent and up-
 
 If you have a Postman account, are not currently part of a paid Postman Pro or Enterprise team, and have never participated in a Postman Pro trial team, you can [delete your account](https://go.postman.co/settings/me).
```
Comment (DannyDainton) [inline legacy-documentation/v5/postman/launching_postman/syncing.md]: Older plan names :(

File: legacy-documentation/v5/postman_for_teams/cloud_api.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ Every user is permitted to create upto 10 keys. To increase this limit, please c
 
 ### Getting Help
 
-For help regarding accessing the Postman API, feel free to discuss it in the `#cloudapi` channel of our [Slack Community][5] ([get invite][6]). You can also drop in a line at [help@getpostman.com](mailto:help@getpostman.com).
+For help regarding accessing the Postman API, feel free to discuss it in the `#cloudapi` channel of our [Slack Community][5] ([get invite][6]). You can also drop in a line at [help@postman.com](mailto:help@postman.com).
```
Comment (DannyDainton) [inline legacy-documentation/v5/postman_for_teams/cloud_api.md]: I don't think we use/monitor the Slack Community space anymore after it was switched to the community.

File: README.md (hunk 1 above)
Comment (SueSmith) [reply README.md]: oops sorting now

File: legacy-documentation/v5/billing_and_payment/pricing-policy.md (hunk 2 above)
Comment (SueSmith) [reply legacy-documentation/v5/billing_and_payment/pricing-policy.md]: we haven't been updating content in legacy docs..

File: legacy-documentation/v5/postman/launching_postman/syncing.md (hunk 3 above)
Comment (SueSmith) [reply legacy-documentation/v5/postman/launching_postman/syncing.md]: again..

File: legacy-documentation/v5/postman_for_teams/cloud_api.md (hunk 4 above)
Comment (SueSmith) [reply legacy-documentation/v5/postman_for_teams/cloud_api.md]: again been leaving legacy docs alone, think we'll remove these from the repo at some point and have some sort of way to view older versions

**Comment by SueSmith** [comment]
@tristandenyer merged, holding off on deleting the branch in case you still need it

**Pull request merged**